mmv1/products/gkebackup/api.yaml

```diff
@@ -190,17 +190,17 @@ objects:
             description: |
               If True, include all namespaced resources.
             exactly_one_of:
-              - all_namespaces
-              - selected_namespaces
-              - selected_applications
+              - backupConfig.0.all_namespaces
+              - backupConfig.0.selected_namespaces
+              - backupConfig.0.selected_applications
```

Comment:
@shuyama1 do these paths look correct?

Reply:
yeah! They look fine and the downstream `ExactlyOneOf` is generated correctly as can be seen at [the diff report](https://github.com/GoogleCloudPlatform/magic-modules/pull/6845#issuecomment-1331410870)